front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -25,6 +25,13 @@ export default function SpeedLimitByTagSelector({
   className = '',
 }: SpeedLimitByTagSelectorProps) {
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
+  const defaultTag = 'MA100';
```

Comment:
I don't think that `MA100` value il related to the component  but rather to the use we make of it.
Maybe you can put it in an external consts file as `DEFAULT_TAG` and import the value. It will be less `hardcoded`

Reply:
Yes, good catch. I will change this 👌

Reply:
done